standalone/src/test/java/io/pravega/local/InProcPravegaClusterTest.java

```diff
@@ -56,6 +56,9 @@ public void setUp() throws Exception {
                                            .enableTls(tlsEnabled)
                                            .certFile("../config/cert.pem")
                                            .keyFile("../config/key.pem")
+                                           .jksKeyFile("../config/bookie.keystore.jks")
```

Comment:
Could we have these files in some `test/resources` folder so that we don't confuse real config files with the ones that applications are supposed to use? Also, this seems to be starting only the client side, where do we start the server in a secure manner, is that in `ZooKeeperServiceRunner`?

Reply:
Yes. `ZooKeeperServiceRunner` starts the server in a secure manner.

Reply:
Ok. What about this:

> Could we have these files in some test/resources folder so that we don't confuse real config files with the ones that applications are supposed to use?

Reply:
Sure. The scope of this is bigger than this PR. Created an issue to track this: https://github.com/pravega/pravega/issues/2984